core/lib/generators/solidus/install/install_generator.rb

```diff
@@ -1,14 +1,17 @@
 # frozen_string_literal: true
 
 require 'rails/generators'
+require 'rails/generators/app_base'
 require 'rails/version'
 require_relative 'install_generator/bundler_context'
 require_relative 'install_generator/support_solidus_frontend_extraction'
 require_relative 'install_generator/install_frontend'
 
 module Solidus
   # @private
-  class InstallGenerator < Rails::Generators::Base
+  class InstallGenerator < Rails::Generators::AppBase
+    argument :app_path, type: :string, default: Rails.root
```

Comment:
The `app_path` line is need in order to provide a default value for the argument. Otherwise, running `solidus:install` would raise the error "No value provided for required arguments 'app_path'":

```
my_app 16:54:55 $ bin/rails generate solidus:install --auto-accept --sample=false --frontend=solidus_frontend --with-authentication
No value provided for required arguments 'app_path'
```

See https://app.circleci.com/pipelines/github/nebulab/solidus/586/workflows/16ee611a-7eb9-48f3-bd39-3a1e189029b2/jobs/7015/parallel-runs/0/steps/0-114 for an instance of this failure.